nexus/src/app/background/init.rs

```diff
@@ -2,7 +2,90 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-//! Specific background task initialization
+//! Initialize Nexus background tasks
```

Comment:
This comment is excellent @davepacheco! Thanks for writing it up  so thoroughly.